src/test/java/org/jitsi/meet/test/base/AbstractParticipantHelper.java

```diff
@@ -60,30 +89,54 @@ public void ensureTwoParticipants()
      */
     public void ensureThreeParticipants()
     {
-        waitForParticipant1ToJoinMUC();
-        waitForSecondParticipantToConnect();
+        waitForParticipant1ToJoinMUC(null, null);
+        waitForSecondParticipantToConnect(null);
         waitForThirdParticipantToConnect();
     }
 
-
     /**
-     * Waits until the owner joins the room, creating and starting the owner
-     * if it hasn't been started.
+     * Creates the first participant (owner) if not already created and
+     * join him in the room.
+     * @param roomParameter an extra parameter to the url. The fragment adds
+     * parameters as # where roomParameter actually changes query parameters
+     * adding ?something=value.
+     * @param fragment adds the given string to the fragment part of the URL
      */
-    private void waitForParticipant1ToJoinMUC()
+    public void startParticipant1(String roomParameter, String fragment)
     {
         if (participant1 == null)
         {
             participant1
                 = ParticipantFactory.getInstance()
-                    .createParticipant("web.participant1");
+                .createParticipant("web.participant1");
         }
 
         if (participant1.isHungUp())
```

Comment:
calling " startParticipant1" method will have no effect if participant is "not hung up" ? it feels weird, if it's expected to be called in some particular state then it should be checked if the assertion is right and if not throw IllegalStateException.

Reply:
Fixed